### PR TITLE
Fixed #28773 -- Forced pot files to use UNIX-style newlines

### PR DESCRIPTION
**Pull request by claudep**
Thanks Hendy Irawan for the analysis and report.

### CODE REVIEW

File: django/core/management/commands/makemessages.py [hunk 1]
```diff
@@ -188,7 +188,9 @@ def write_pot_file(potfile, msgs):
                 header_read = True
             lines.append(line)
     msgs = '\n'.join(lines)
-    with open(potfile, 'a', encoding='utf-8') as fp:
+    # Forcing POT files newline because of a gettext bug:
```
Comment (timgraham) [inline django/core/management/commands/makemessages.py]: suggested comment: `# Force newlines of POT files to '\n' to work around https://savannah.gnu.org/bugs/index.php?52395`

**Comment by claudep** [comment]
Tim, are you OK for a 2.0 backport?

**Comment by timgraham** [comment]
Ok

**Comment by helio-correia** [comment]
Tim, are you OK for a 1.11 backport? xD

**Comment by timgraham** [comment]
Per our [supported versions policy](https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions), 1.11 is only receiving fixes for security and data loss issues.

**Comment by helio-correia** [comment]
Sorry my bad and thanks for reply

**Pull request merged**